cloth-diapers/cloth-diapers-basics.md

```diff
@@ -0,0 +1,296 @@
+so, I think this is a good investment because it lets you learn the ropes and
+they hold their value extremely well (more so than OS) so can be sold on easily.
+
+
+## Diaper Types
+Now that we have 'dialed things in' a little more, there are three types of
+diapers we use (there are others, but these are the only 3 in our rotation):
+
+### 1. All-In-Ones (AIO)
+This is our 'favourite' kind of diaper because it requires the **_least_ time
+to put together** by _far_. For the majority you just have to push the 'tongue'
+(sewn in insert) back into the pocket and that's it.
+With a small number (in our stash it's the Motherease Wizard Unos and
+Bellz Bumz Reusabelles), they're ready to go straight off the line.
+
+Whilst we now find we have to reinforce the majority of these with an extra
+3 layer bamboo insert, this isn't the case with all of them.
+
+**The biggest issue with AIOs** is that they take a lot longer to dry.
+
+**The key thing to be aware of** is that not all AIOs are created equally
+in terms of absorption. It is _very important_ to understand what the material
+composition of these diapers is so that you know what you're dealing with.
+
+For example:
+[Modern Cloth Nappies AIOs](https://www.modernclothnappies.co.uk/collections/pearl-pocket-all-in-one-nappies/products/pearl-pocket-all-in-one-reusable-one-size-nappy)
+(my absolute favourite AIOs from our stash), have a super absorbent
+hemp cotton mix insert.
+<img width="533" alt="image" src="https://user-images.githubusercontent.com/4185328/170531761-98c62f2c-a3ac-4eab-b941-04940c289e2f.png">
+
+The [Bambino Miosolo AIOs](https://www.bambinomio.com/collections/miosolo/products/miosolo-diaper)
+sit at the complete opposite end of that scale, with a microfibre core which,
+despite the claims on the website of 'super absorbency', has _very little
+absorbency_ and _very good marketing_.
+
+I wouldn't risk using one of these without reinforcement at any time of day.
+Why do we still have them you ask? Because they have a _great_ fit and are
+very practical in that sense!
+
+<img width="492" alt="image" src="https://user-images.githubusercontent.com/4185328/170533205-f0cfd3c2-b9f6-49a2-8f9c-702ea9f271cf.png">
+
+
+### 2. Pocket Diapers
+We started using One Size diapers in the Spring and AIOs were a firm favourite
+because as I mentioned, the gave me very little work to put together.
+
+But then Winter came [insert your favourite Game of Thrones meme here].
+
+I struggled to get anything to dry, even with a dehumidifier on constantly.
+So we moved to pocket diapers for Winter.
+
+They take a LOT more time to put together but the shells air dry very quickly
+and if you're desperate you can throw the inserts in the tumble dryer too.
+
+We use _high quality_ natural material inserts.
+
+Our ***hemp inserts*** are triple layer hemp/cotton inserts with a _high_ percentage
+of hemp (55% hemp, 45% cotton): https://www.eco-tiny.com/product/absorvente-de-canhamo-e-algodao-organico-ecotiny
```

Comment:
Eco-tiny is very PT-focussed. 
![image](https://user-images.githubusercontent.com/194400/173179997-95e53024-7dd2-4379-9375-73138911f6f9.png)
they have a language-switcher at the top but don't appear to ship beyond PT ...

Consider having a "where to buy" section with both _local_ and international links. ideally affiliate links so you can get paid for all your research ... 🔗

Reply:
I agree. But this is the section about specifically what _we_ use and we use these inserts because they are a LOT better than the 5 or 6 other brands I tried so I have to reference these specifically.

I'll add an issue for a Where to Buy section. 👍 #37